swagger_server/itm/itm_scenario_session.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
         self.first_answer: bool = True
         # hacky stuff for adept
         self.casualty_ids = []
-        self.probes_responded_to = []
+        self.adept_evac_happened = False
```

Comment:
This is fine for the September deliverable, in the future, I think we want to name this to a generic variable name so we can use it for anything that might hold a scenario from having end_scenario available.